glm/detail/func_common.inl

```diff
@@ -91,7 +91,7 @@ namespace detail
 	template<length_t L, typename T, qualifier Q, bool Aligned>
 	struct compute_mix_vector<L, T, bool, Q, Aligned>
 	{
-		GLM_FUNC_QUALIFIER static vec<L, T, Q> call(vec<L, T, Q> const& x, vec<L, T, Q> const& y, vec<L, bool, Q> const& a)
+		GLM_FUNC_QUALIFIER GLM_CONSTEXPR static vec<L, T, Q> call(vec<L, T, Q> const& x, vec<L, T, Q> const& y, vec<L, bool, Q> const& a)
 		{
 			vec<L, T, Q> Result;
```

Comment:
This seems to be failing the GCC6 C++14 builds in Travis. I think it's the C++14 vs C++17 discrepancy in what's allowed inside `constexpr` functions. This StackOverflow issue sheds some extra light: https://stackoverflow.com/questions/41618576/what-is-allowed-in-a-constexpr-function

Reply:
Yes, I've seen it. My bad. I'll fix it this evening.